ca/ca.go

```diff
@@ -263,9 +285,22 @@ func (ca *CAImpl) CompleteOrder(order *core.Order) {
 	order.Unlock()
 }
 
-func (ca *CAImpl) GetRootCert() *core.Certificate {
-	if ca.root == nil {
+func (ca *CAImpl) GetRootCert(no *int) *core.Certificate {
```

Comment:
It seems strange to make the `int` parameter a pointer type. Was there a specific reason you did this I'm missing?

Otherwise I think it would be better as `GetRootCert(no uint)`. That will also let you remove the `if no == nil` check. Changing the `int` to `uint` will also let you drop the `if *no < 0` check.

Reply:
My idea was that `nil` stands for "default chain", and non-`nil` for Alternative #`no`. But it is probably easier to understand if `no == 0` is the "default chain", and `no == 1` up to `no == PEBBLE_ALTERNATE_ROOTS` the alternative chains.

Reply:
I think `no == 0` as the default chain will be more idiomatic.

Reply:
Good, then it should be fine now :)